src/SFA.DAS.EAS.Web.IntegrationTests/TestUtils/ApiTester/ApiIntegrationTester.cs

```diff
@@ -268,5 +256,35 @@ private bool IsAcceptableStatusCode(CallResponse response, IEnumerable<HttpStatu
         {
             return acceptableStatusCodes == null || acceptableStatusCodes.Contains(response.Response.StatusCode);
         }
+
+        private EmployerAccountsDbBuilder GetInstanceOfEmployerAccountsDbBuilderWithTransaction(IContainer container)
+        {
+            var _ = container.GetInstance<EmployerAccountsDbBuilder>();
+            EmployerAccountsDbBuilderExtension.EnsureTransaction(_);
```

Comment:
minor comment: I'm not sure why we need extensions to classes that we have code ownership for and are private to this class library.

Reply:
Missed refactoring this from earlier spike it is on the instance now.